### PR TITLE
Resource Timing: encoded/decoded body size for SW responses

### PR DESCRIPTION
**Pull request by noamr**

### CODE REVIEW

File: service-workers/service-worker/resource-timing-bodySize.https.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        const uid = token();
+        const worker_url = `resources/fetch-response.js?uid=${uid}`;
+        const scope = `resources/fetch-response.html?uid=${uid}`;
+        const iframe = document.createElement('iframe');
+        iframe.src = `${scope}&path=${name}`;
+        const registration = await service_worker_unregister_and_register(t, worker_url, scope);
+        t.add_cleanup(() => registration.unregister());
+        t.add_cleanup(() => iframe.remove());
+        await wait_for_state(t, registration.installing, 'activated');
+        const waitForMessage = new Promise(resolve => window.addEventListener('message', ({data}) => resolve(data)));
```
Comment (yoavweiss) [inline service-workers/service-worker/resource-timing-bodySize.https.html:20]: The multiple uses of data here are a bit confusing. Can you rename the "internal" data to buffer, or something similar?

**Comment by yoavweiss** [comment]
Looks good, but as it seems like all implementations are currently failing these tests, I'd somewhat prefer to submit these through an implementation patch to test the tests :)

**Comment by noamr** [comment]
> Looks good, but as it seems like all implementations are currently failing these tests, I'd somewhat prefer to submit these through an implementation patch to test the tests :)

Fair enough, on its way

**Comment by noamr** [comment]
Added tests via chromium

**Pull request closed without merging**